lib/Lemplate.pm

```diff
@@ -538,6 +534,39 @@ local function stash_set(stash, k, v, default)
     end
 end
 
+_M.vmethods = {
+    join = function(list, delim)
+        if delim == nil then
+            delim = ' '
+        end
+        local out = {}
+        for i = 1, #list, 1 do
+            table.insert(out, list[i])
```

Comment:
Agreed. We should avoid using `table.insert` since it is an `O(n)` operation due to the `lj_tab_len` call inside its implementation.